src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -7374,13 +7377,7 @@ namespace WaterThermalTanks {
 			}
 
 			//   Simulate tank if HP compressor unavailable for water heating
-			{ auto const SELECT_CASE_var( HeatPump.TankTypeNum );
-
-			if ( SELECT_CASE_var == MixedWaterHeater ) {
-				CalcWaterThermalTankMixed( WaterThermalTankNum );
-			} else if ( SELECT_CASE_var == ( StratifiedWaterHeater ) ) {
-				CalcWaterThermalTankStratified( WaterThermalTankNum, HPPartLoadRatio );
-			}}
+			CalcWaterThermalTank( WaterThermalTankNum );
```

Comment:
This was being done enough times that I created a helper function to make this cleaner.